qabelbox/src/main/java/de/qabel/qabelbox/activities/CreateIdentityActivity.java

```diff
@@ -40,6 +41,13 @@
 public class CreateIdentityActivity extends BaseWizardActivity {
 
     public static final int REQUEST_CODE_IMPORT_IDENTITY = 1;
+    /**
+     * Fake the prefix request
+     *
+     * This is set by the QblJUnitTestRunner to prevent network requests to the block server
+     * in test runs.
+     */
+    public static boolean FAKE_COMMUNICATION = false;
```

Comment:
Hmm should not be here. We could create a FakePrefixServer and inject it or override the loadPrefixInBackground function in a TestCreateIdentityActivity